core/backend/src/BackendHubAccess.ts

```diff
@@ -186,22 +270,16 @@ export interface BackendHubAccess {
   /** get an array of the briefcases assigned to a user. */
   getMyBriefcaseIds: (arg: IModelIdArg) => Promise<BriefcaseId[]>;
 
-  /**
-   * download a v1 checkpoint
-   * @internal
-   */
-  downloadV1Checkpoint: (arg: CheckpointArg) => Promise<ChangesetIndexAndId>;
+  /** Download a v1 checkpoint. */
+  downloadV1Checkpoint: (arg: DownloadCheckpointArg) => Promise<ChangesetIndexAndId>;
```

Comment:
Do we really want to move it to @beta?

Reply:
If we announce it as 3.3.0 feature, it should be. If not, we can keep it @internal at least till `BackendIModelAccess` implementation is updated.

@calebmshafer, am I misremembering or did you mention Progress API should be at least in @beta ?

Reply:
Mainly my uncertainty comes from naming: `v1` vs `v2` and whether that implementation detail should be made public.. Maybe we could only expose `downloadCheckpoint`?

Reply:
This method should be marked deprecated and soon removed. Downloading of checkpoints should *not* be a part of BackendHubAccess at all, and i have removed it in my "daemonless" branch. 

Downloading of checkpoints can only be done in core-backend. All `BackendHubAccess` does is give the checkpointProps.